src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
```

Comment:
### Line 23
## CodeGuard Suggestions
**Suggestion:** Extract the json from the response using a helper method
**Reason:** To make the code more readable, maintainable and secure